graph_nets/demos_tf2/shortest_path.ipynb

```diff
@@ -408,15 +439,18 @@
     "    target_ph: The target graph's placeholders, as a graph namedtuple.\n",
     "\n",
     "  Returns:\n",
-    "    feed_dict: The feed `dict` of input and target placeholders and data.\n",
+    "    input_graphs: The instance of a Graph input.\n",
+    "    target_graphs: The instance of a Graph target.\n",
     "    raw_graphs: The `dict` of raw networkx graphs.\n",
     "  \"\"\"\n",
     "  inputs, targets, raw_graphs = generate_networkx_graphs(\n",
     "      rand, batch_size, num_nodes_min_max, theta)\n",
-    "  input_graphs = utils_np.networkxs_to_graphs_tuple(inputs)\n",
-    "  target_graphs = utils_np.networkxs_to_graphs_tuple(targets)\n",
-    "  feed_dict = {input_ph: input_graphs, target_ph: target_graphs}\n",
-    "  return feed_dict, raw_graphs\n",
+    "  input_graphs = networkxs_to_tf_graphs_tuple(inputs)\n",
+    "  target_graphs = networkxs_to_tf_graphs_tuple(targets)\n",
+    "  \n",
+    "  return input_graphs,target_graphs, raw_graphs\n",
```

Comment:
Leave space after commas, it would be great if you could run a linter on this (but make sure it leaves the two space indentation).